bin/show_org_stats.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :since, "Since what date.", :type => :string, :required => true
+
+  ManageIQ::Release.common_options(self, :except => :dry_run)
+end
+
+class OrgStats
+  def initialize(since:, **opts)
+    @since = since
+    @opts  = opts
+
+    @total_commits = Hash.new(0)
+    @total_merges  = Hash.new(0)
+    @names = {}
+  end
+
+  def run
+    ManageIQ::Release.each_repo(**@opts) { |repo| run_one(repo) }
+
+    puts
+    puts "Total Commits:"
+    print_totals(@total_commits)
+
+    puts
+    puts "Grand Total Commits: #{@total_commits.values.sum}"
+
+    puts
+    puts "Total Merges:"
+    print_totals(@total_merges)
+
+    puts
+    puts "Grand Total Merges: #{@total_merges.values.sum}"
```

Comment:
not necessarily for this PR, but would be nice to have similar stats on Issues Opened and Closed